### PR TITLE
Adding School Model.

### LINKED ISSUE #26: Add School Model
The `School` model should have:
- [x] **Name**
- [x] **Location** (FK to Location)
- [x] **Type** (Enum: `Boarding`, `Charter`, `Language Immersion`, `Magnet`, `Montessori`, `Online`, `Parochial`, `Private Special Education`, `Private`, `Public`, `Reggio Emilia`, `Religious`, `Waldorf`. *Default*: `Public`)
- [ ] **Grades** (For example: PK and K-8)
- [x] **Website** (I.e. [Von Linne](http://www.linneschool.org/))
- [x] **URL to School Details** (I.e. [Von Linne](http://schoolinfo.cps.edu/schoolprofile/schooldetails.aspx?SchoolId=610039))
- [ ] **Principal** (FK to Contact?)

Relies on #21

### PR DESCRIPTION
**Pull request by karbassi**
Fixes #26

### CODE REVIEW

File: web/location/admin.py [hunk 1]
```diff
@@ -48,3 +49,24 @@ class LocationAdmin(admin.ModelAdmin):
...
+    '''Admin View for School'''
+
+    list_display = (
+        'id',
+        'name',
+        'school_type',
+    )
+    list_filter = (
+        'school_type',
+        # ('school_type', ChoiceDropdownFilter),
```
Comment (danmconrad) [inline web/location/admin.py]: Need this?

**Comment by karbassi** [comment]
@danmconrad Can you test with the changes I've pushed now.

**Comment by danmconrad** [comment]
This looks great! Everything runs just fine on my machine. Where did you get the list of schools from? I'm guessing it's not a full list since I can't find my grade schools (Kilmer, Eugene Field, Franklin Fine Arts Center).

**Comment by danmconrad** [comment]
I found out how to fix the Heroku build- when adding a git dependency to pipenv, you need run:
```bash
pipenv install -e git+git://github.com/karbassi/django-address#egg=address
```
Where the egg is the name of the import for the package. I'd suggest we do this after we merge this PR.

**Pull request merged**